mlir/lib/Quantum/Transforms/MergeRotationsPatterns.cpp

```diff
@@ -21,13 +21,16 @@
 #include "llvm/ADT/StringSet.h"
 #include "llvm/Support/Debug.h"
 #include "llvm/Support/Errc.h"
+#include "mlir/Dialect/Math/IR/Math.h"
+#include "mlir/Dialect/Arith/IR/Arith.h""
```

Comment:
I think there's an extra quotation mark here, which made all your code into comments below :)

Reply:
(Also this file is already included in here no?)

Reply:
Thank you, Done!